examples/fluid/sphere_surface_tension_2d.jl

```diff
@@ -1,28 +1,28 @@
 # In this example we can observe that the `SurfaceTensionAkinci` surface tension model correctly leads to a
-# surface minimization of the water square and approaches a circle.
+# surface minimization of the water square and approaches a sphere.
 using TrixiParticles
 using OrdinaryDiffEq
 
 fluid_density = 1000.0
 
 particle_spacing = 0.1
+square_size = 0.5
 
-sound_speed = 20
+sound_speed = 20.0
 state_equation = StateEquationCole(; sound_speed, reference_density=fluid_density,
                                    exponent=7, clip_negative_pressure=true)
 
-particle_spacing = 0.1
-
-# for all surface tension simulations needs to be smoothing_length = 4r
-smoothing_length = 2.0 * particle_spacing
-smoothing_kernel = WendlandC2Kernel{2}()
-nu = 0.01
-
+# For all surface tension simulations, we need a compact support of `2 * particle_spacing`
 # smoothing_length = 2.0 * particle_spacing
-# smoothing_kernel = SchoenbergCubicSplineKernel{2}()
-# nu = 0.025
-
-fluid = RectangularShape(particle_spacing, (5, 5), (0.0, 0.0),
+# smoothing_kernel = WendlandC2Kernel{2}()
+# nu = 0.01
+
+smoothing_length = 1.0 * particle_spacing
+smoothing_kernel = SchoenbergCubicSplineKernel{2}()
+nu = 0.025
+no_square_particles = round(Int, square_size / particle_spacing)
```

Comment:
```suggestion

n_particles_per_dimension = round.(Int, fluid_size ./ particle_spacing)
```
Same variable name as used internally.